modules/standard/Reflection.chpl

```diff
@@ -141,12 +141,24 @@ proc canResolve(param fname : string, args ...) param : bool
 proc canResolveMethod(obj, param fname : string) param : bool
   return __primitive("method call resolves", obj, fname);
 
-/* Returns true if a method on `obj` named `fname` taking the arguments in args
+/* Returns true if a method on `obj` named `fname` taking the arguments in `args`
    could be called in the current scope.
    */
 proc canResolveMethod(obj, param fname : string, args ...) param : bool
   return __primitive("method call resolves", obj, fname, (...args));
 
+/* Returns true if a method on type `t` named `fname` taking in no
```

Comment:
Is there a way to adjust this documentation comment to use the words "type method" so that it's clearer that is what this method is about?